src/utils/helpers/review/listReviewerResponses.ts

```diff
@@ -12,21 +12,20 @@ const listReviewResponses = ({ userId, reviewSections }: ReviewerResponsesPayloa
   const assignedSections = Object.values(reviewSections).filter(
```

Comment:
This file should be called `listReviewResponses.ts`, shouldn't it?

Reply:
Sure. Thanks